packages/core/strapi/lib/utils/startup-logger.js

```diff
@@ -19,15 +18,13 @@ module.exports = (app) => {
         chars: { mid: '', 'left-mid': '', 'mid-mid': '', 'right-mid': '' },
       });
 
-      const isEE = app.EE === true && ee.isEE === true;
```

Comment:
That's what I understood as well, but I added [this](https://github.com/strapi/strapi/blob/ecc455182c57c5f31413a95f86f19dc3840e867d/packages/core/strapi/lib/Strapi.js#L125) to prevent tampering with `strapi.EE`.
Maybe there are other ways to tamper with it that I'm unaware of?